Controllers/TransactionController.cs

```diff
@@ -0,0 +1,27 @@
+using baseledger_replicator.BusinessLogic.Transactions.Queries;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+
+namespace baseledger_replicator.Controllers;
+
+[ApiController]
+[Authorize]
+[Route("api/[controller]")]
+public class TransactionController : BaseController
+{
+    public TransactionController()
+    {
+
+    }
+
+    /// <summary>
+    /// Retrieves transaction by txHash
+    /// </summary>
+    /// <param name="txHash">Transaction hash</param>
+    [HttpGet("transaction/{txHash}")]
```

Comment:
no need for transaction in the route here, it is already in the route based on line 9